spec/features/visitors/email_confirmation_spec.rb

```diff
@@ -0,0 +1,132 @@
+require 'rails_helper'
+
+feature 'Email confirmation during sign up' do
+  scenario 'confirms valid email and sets valid password' do
+    email = 'test@example.com'
+    sign_up_with(email)
+    raw_confirmation_token, = Devise.token_generator.generate(User, :confirmation_token)
+    User.find_by(email: email).update(
+      confirmation_token: raw_confirmation_token, confirmation_sent_at: Time.current
+    )
+
+    visit "/users/confirmation?confirmation_token=#{raw_confirmation_token}"
+
```

Comment:
As far as I know, we do not have test that opens the email and clicks. Good call.

Is there an easy way to do that without bringing in an external lib? Was just looking into https://github.com/DockYard/capybara-email but would be nice to do this without adding a dependency